include/rta/fullsupernodelist.h

```diff
@@ -0,0 +1,144 @@
+#ifndef FULLSUPERNODELIST_H
+#define FULLSUPERNODELIST_H
+
+#include <rta/supernode.h>
+#include <rta/DaemonRpcClient.h>
+#include <cryptonote_config.h>
```

Comment:
Why are you use standard header includes for local headers?

Reply:
fixed with 48dcf0b32b60a36f5223ab7e0987a071daf7f0cc